be-rust/src/commonMain/resources/lang/temper/be/rust/temper-core/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.6.0"
 license = "Apache-2.0 OR MIT"
 edition = "2021"
 rust-version = "1.71.1"
+readme = false
```

Comment:
[Here `false` means no readme.](https://rustwiki.org/en/cargo/reference/manifest.html#the-readme-field) The [crates.io page](https://crates.io/crates/temper-core/0.6.0) would look much better with a readme, though. We have one for py temper-core, though small. Maybe could centralize this.

And for temper-built libs, maybe can standardize forwarding README.md files or manufacturing small ones from other metadata when no readme is present. But I didn't feel like going that deep here.

And I also don't generate either readme (not even to false in this pr) or documentation for temper-built libs yet at all. They aren't required for publishing.